ompi/include/mpif-values.pl

```diff
@@ -216,8 +216,8 @@ sub write_file {
 
 my $constants;
 
-$constants->{MPI_VERSION} = 3;
-$constants->{MPI_SUBVERSION} = 1;
+$constants->{MPI_VERSION} = `grep mpi_standard_version VERSION | cut -d '=' -f 2`;
```

Comment:
It feels like it would be better to use native perl here?

Reply:
Agreed. Fixed in the latest push.